tests/test_daemon.cpp

```diff
@@ -643,6 +643,14 @@ TEST_P(DaemonCreateLaunchTestSuite, default_cloud_init_grows_root_fs)
     auto mock_factory = use_a_mock_vm_factory();
     mp::Daemon daemon{config_builder.build()};
 
+    auto mock_factory_scope = mpt::MockProcessFactory::Inject();
+
+    mock_factory_scope->register_callback([&](mpt::MockProcess* process) {
+        const mp::ProcessState qemuimg_exit_status{0, mp::nullopt};
+        const QByteArray qemuimg_output(fake_img_info(mp::MemorySize{("1048577")}));
+        simulate_qemuimg_info(process, qemuimg_exit_status, qemuimg_output);
+    });
```

Comment:
Could this be refactored into something shared between the tests here?

Reply:
Good idea, done.